server/main.lua

```diff
@@ -183,35 +183,25 @@ function UpdateGrade(src, data, type)
     end
 end
 
----@param src number
----@return table
-function GetPlayers(src)
+-- Get closest player sv
+lib.callback.register('qb-bossmenu:getplayers', function(_, closePlayers)
 	local players = {}
-	local playerPed = GetPlayerPed(src)
-	local pCoords = GetEntityCoords(playerPed)
-	for _, v in pairs(exports.qbx_core:GetPlayers()) do
-		local targetped = GetPlayerPed(v)
-		local tCoords = GetEntityCoords(targetped)
-		local dist = #(pCoords - tCoords)
-		if playerPed ~= targetped and dist < 10 then
-			local ped = exports.qbx_core:GetPlayer(tonumber(v))
-			players[#players + 1] = {
-				id = v,
-				coords = GetEntityCoords(targetped),
-				name = ped.PlayerData.charinfo.firstname .. " " .. ped.PlayerData.charinfo.lastname,
-				citizenid = ped.PlayerData.citizenid,
-				sources = GetPlayerPed(ped.PlayerData.source),
-				sourceplayer = ped.PlayerData.source
-			}
-		end
+	for _, v in pairs(closePlayers) do
+		local ped = exports.qbx_core:GetPlayer(v)
```

Comment:
```suggestion
		local player = exports.qbx_core:GetPlayer(v)
```
A ped is a human/animal GTA object. What you are getting here is the qbx_core player object. They aren't the same thing.

Reply:
Fixed